evaluation/benchmarks/max-temp/max-temp-preprocess.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
-sed 's;^;http://ndr.md/data/noaa/;' |
+sed 's;^;atlas-group.cs.brown.edu/data/noaa/;' |
```

Comment:
these changes were in a different PR, why are they visible here?

Reply:
can you rebase your branch to the correct one (or cherrypick your commits only?)

Reply:
Yes, I'll separate the changes corresponding to benchmark updates to its own PR!